RangeError and TypeError are JS exception; SyntaxError normally not (#15438)
mdn/content: files/en-us/web/api/mediacapabilities/encodinginfo/index.md

### Exceptions

- {{jsxref("TypeError")}}

>   - : Thrown if the `MediaConfiguration` passed to the `encodingInfo()` method is invalid,
>     either because the type is not video or audio, the `contentType` is not a valid codec MIME type,
>     or any other error in the media configuration passed to the method,
>     including omitting any of the [media encoding configuration](/en-US/docs/Web/API/MediaEncodingConfiguration) elements.

Reviewer: ```suggestion
  - : Thrown if the `MediaConfiguration` passed to the `encodingInfo()` method is invalid, which may be for any of the following reasons:
    - the type is not video or audio
    - the `contentType` is not a valid codec MIME type,
    - there is some other error in the media configuration passed to the method, including omitting any of the [media encoding configuration](/en-US/docs/Web/API/MediaEncodingConfiguration) elements.
```

(or something. Let's make this long sentence easier to scan.)
